pkg/providers/subnet/suite_test.go

```diff
@@ -103,10 +104,12 @@ var _ = Describe("SubnetProvider", func() {
 			}
 			subnets, err := awsEnv.SubnetProvider.List(ctx, nodeClass)
 			Expect(err).To(BeNil())
+			fmt.Printf("subnets: %v\n", subnets)
```

Comment:
Can drop this now

Reply:
That's my bad, missed dropping this in my latest commit. Will remove.